.github/workflows/nightly.yml

```diff
@@ -0,0 +1,18 @@
+name: CI Checks
+
+on:
+  schedule:
+  #- cron: "0 0 * * *"
+jobs:
+  formatting:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build nightly
+        run: ./scripts/build-nightly
+      - name: Publish
+        run: npm publish
```

Comment:
Do we want to publish version under a `nightly` flag?

```
npm publish --tag nightly
```

https://docs.npmjs.com/adding-dist-tags-to-packages

Reply:
Good idea.  Done.  What was the second question?